README.rst

```diff
@@ -76,17 +76,35 @@ Medallion provides a command-line interface to start the TAXII Server
       --log-level {DEBUG,INFO,WARN,ERROR,CRITICAL}
                             The logging output level for medallion.
 
+      -c CONF_FILE, --conf-file CONF_FILE
+                            Path to a single configuration file. Defaults to the
+                            value of the MEDALLION_CONFFILE environment variable
+                            or /etc/xdg/medallion/3/medallion.conf.
+
+      --conf-dir CONF_DIR   Path a directory containing JSON configuration files
```

Comment:
"Path a directory" should be "Path to a directory"

Reply:
fixed here and also in the actual parser help text